### DISCUSSION TITLE
Hice el diagrama de clases en Lucidchart

### DISCUSSION THREAD

**Discussion by ezecastiglione18** [dds-jv/foro#211] [Consultas generales] [answered]
Buenas! Tengo una consulta: hice el diagrama de clases en Lucidchart, pero en el formulario de respuesta solamente esta la opcion para copiar el link del repo donde esta la resolucion. Podrian agregar una opcion para que tambien podamos subir archivos? O como podriamos hacer?

Saludos!

**Comment by gastonprieto** [comment, marked as answer]
buenas @ezecastiglione18 

Podrias subir el archivo a un repo, podrías tener un repo para todas las entregas, o usar uno distinto por entregas, como prefieras
